### PR TITLE
Initial PR. Refactored prototype assignment from JavaScript-III.

### PR DESCRIPTION
**Pull request by catslanderer**

### CODE REVIEW

File: assignments/lambda-classes.js [hunk 1]
```diff
@@ -1 +1,180 @@
...
+    adjustment *= Math.floor(Math.random() * 2) === 1 ? 1 : -1; // Randomly determines whether the grade adjustment will be positive or negative
+    student.grade += adjustment; // Applies the grade adjustment
+    if (student.grade < 1) {
+      student.grade = 1; // Stops the student's grade from being lower than 1
+    }
+    if (student.grade > 100) {
+      student.grade = 100; // Stops the student's grade from being higher than 100
+    }
+  }
+}
```
Comment (DaftBeowulf) [inline assignments/lambda-classes.js:39]: nicely done with this method! good accounting for edge cases as well

**Review by DaftBeowulf — COMMENTED** [review]
very well done with refactoring into ES6 class syntax, and getting MVP and stretch with your grade and graduate methods! you have a methodical approach with your if/else statements in those methods which shows that you're cognizant of possible edge cases, and you have thorough console.logs to test and show why you inheritance chains work as intended. great job, this practice with class syntax will  set you up well for working with React
